ansible_collections/arista/avd/roles/eos_designs/python_modules/connected_endpoints/utils.py

```diff
@@ -57,6 +57,12 @@ def _filtered_connected_endpoints(self) -> list:
                             " Notice that some or all of these variables could be inherited from 'port_profiles'"
                         )
 
+                    # Check flow tracking variables are properly defined
```

Comment:
Feels weird to have this verification here, when everything is not parsed until we render. NMaybe it makes sense to keep it here, but then make sure to make it as efficient as possible.

Reply:
Yes, I have placed it here, because this way we can give the proper context path. 
We could also pass context path everywhere but that'd be too cumbersome. 
We are already iterating over the config here, hence just adding 1 more if condition feels ok to me.